### DISCUSSION TITLE
3.0.3

### DISCUSSION THREAD

**Discussion by zHaytam** [Blazor-Diagrams/Blazor.Diagrams#488] [Announcements]
### Added

- Support for .NET 9 (thanks to @[PanayiotisTheodosiou](https://github.com/PanayiotisTheodosiou))
- `registerDefaultBehaviors` argument to `BlazorDiagram` constructor, it's `true` by default and can be disabled if you want to control what behaviors are added and their order.

### Changed

- Make `DiagramCanvas` await un-subscription to resizes subscription (thanks to @[toluwtglobal](https://github.com/toluwtglobal))

### Fixed

- Exception thrown when `node.Visible` was `false` from the creation and the library was trying to observe its resizes when the element won't exist (fixes #476).

### Removed

- gzip versions of assets, it was causing an issue with .NET 9 (fixes #471, thanks a lot @[robertmclaws](https://github.com/robertmclaws))

<hr /><em>This discussion was created from the release <a href='https://github.com/Blazor-Diagrams/Blazor.Diagrams/releases/tag/3.0.3'>3.0.3</a>.</em>

**Comment by fkmbkk2** [comment]
I couldn't get this version to run on .net 9...Any samples out there ?
